specification/policyinsights/resource-manager/readme.md

```diff
@@ -81,6 +81,21 @@ directive:
 
 ```
 
+### Tag: package-2021-04
+
+These settings apply only when `--tag=package-2021-04` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-04'
```

Comment:
I don't think you need this new tag. If you see, the input files are exactly the same as in the previous package. We only add a new package tag if there are changes in the files we want included in a package